### DISCUSSION TITLE
Eligibility criteria

### DISCUSSION THREAD

**Discussion by glennawatson** [dotnet-foundation/Home#54] [General Discussions]
https://github.com/dotnet-foundation/projects#eligibility-criteria

Recently there has been a lot of discussion about new eligibility criteria.

What works in the eligibility criteria and what doesn't?

How would we enforce them?

Could automatic opt-in tools help with eligibility criteria?

How do we get project maintainers buy-in so that eligibility criteria aren't such a surprise in the future?

**Comment by gortok** [comment]
@glennawatson what makes you think eligibility criteria are a surprise?

**Reply by glennawatson** [reply]
Just lack of notification before it happened, only if you get notifications for that repository would you know that it has changed.
